rpc/src/main/proto/scalardb.proto

```diff
@@ -339,3 +339,92 @@ message AbortRequest {
 message AbortResponse {
   TransactionState state = 1;
 }
+
+// TwoPhaseCommitTransaction
+
+service TwoPhaseCommitTransaction {
+  rpc TwoPhaseCommitTransaction(stream TwoPhaseCommitTransactionRequest) returns (stream TwoPhaseCommitTransactionResponse) {
+  }
+  rpc GetState(GetTransactionStateRequest) returns (GetTransactionStateResponse) {
+  }
+  rpc Abort(AbortRequest) returns (AbortResponse) {
```

Comment:
I'm sorry I didn't notice it last time, but what is the reason for having `Abort` rpc separately from `TwoPhaseCommitTransaction`?

Reply:
I think that's the same reason why we have the `Abort` method in `DistributedTransaction`:
https://github.com/scalar-labs/scalardb/pull/238